zeebe/gateway-protocol/src/main/proto/v2/conditionals.yaml

```
@@ -0,0 +1,116 @@
+                summary: Evaluate root-level conditional start events for a specific process definition
+                value:
+                  processDefinitionKey: "2251799813686749"
+                  variables:
+                    orderAmount: 1000
+              Evaluate conditionals for specific tenant:
+                summary: Evaluate root-level conditional start events for process definitions linked to a specific tenant
+                value:
+                  tenantId: "tenant-123"
+                  variables:
+                    orderAmount: 1000
+      responses:
+        "200":
+          description: Successfully evaluated root-level conditional start events.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/EvaluateConditionalResult'
+        "400":
+          $ref: 'common-responses.yaml#/components/responses/InvalidData'
+        "403":
+          description: |
+            The client is not authorized to start process instances for the specified process definition.
+            If a processDefinitionKey is not provided, this indicates that the client is not authorized
+            to start process instances for at least one of the matched process definitions.
+          content:
+            application/problem+json:
+              schema:
+                $ref: 'common-responses.yaml#/components/responses/Forbidden'
+        "404":
+          description: The process definition was not found for the given processDefinitionKey.
+          content:
+            application/problem+json:
+              schema:
+                $ref: 'problem-detail.yaml#/components/schemas/ProblemDetail'
+        "500":
+          $ref: 'common-responses.yaml#/components/responses/InternalServerError'
+        "503":
+          $ref: 'common-responses.yaml#/components/responses/ServiceUnavailable'
+
+components:
+  schemas:
+    ConditionalEvaluationInstruction:
+      type: object
+      required:
+        - variables
+      properties:
+        tenantId:
+          allOf:
+            - $ref: 'identifiers.yaml#/components/schemas/TenantId'
+          description: |
+            Used to evaluate root-level conditional start events for a tenant with the given ID.
+            This will only evaluate root-level conditional start events of process definitions which belong to the tenant.
+        processDefinitionKey:
+          allOf:
+            - $ref: 'keys.yaml#/components/schemas/ProcessDefinitionKey'
+          description: |
+            Used to evaluate root-level conditional start events of the process definition with the given key.
+        variables:
+          type: object
+          description: |
+            JSON object representing the variables to use for evaluation.
+            Variables will be used for evaluation of the conditions and are passed to the process instances which have been triggered.
```

🔧 I guess we can streamline this a bit.
```suggestion
            JSON object representing the variables to use for evaluation of the conditions and to pass to the process instances that have been triggered.
```